src/e2e/java/teammates/e2e/pageobjects/StudentProfilePage.java

```diff
@@ -159,9 +159,13 @@ public void showPictureEditor() {
         waitForUploadEditModalVisible();
     }
 
-    public void verifyPhotoSize(String height, String width) {
-        assertEquals(height, browser.driver.findElement(By.className("profile-pic")).getCssValue("height"));
-        assertEquals(width, browser.driver.findElement(By.className("profile-pic")).getCssValue("width"));
+    public void verifyPhotoSize(int height, int width) {
+        String browserHeight = browser.driver.findElement(By.className("profile-pic")).getCssValue("height");
+        float imageHeight = Float.parseFloat(browserHeight.substring(0, browserHeight.length() - 2));
+        String browserWidth = browser.driver.findElement(By.className("profile-pic")).getCssValue("width");
+        float imageWidth = Float.parseFloat(browserWidth.substring(0, browserWidth.length() - 2));
```

Comment:
`browserWidth.substring(0, browserWidth.length() - 2)` is a magic operation and `2` in it is a magic number. I know the intention here, not necessarily everyone else. At the very least where `2` comes from should be made clearer.